spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/data/neo4j/Neo4jProperties.java

```diff
@@ -142,4 +156,55 @@ public void setEnabled(boolean enabled) {
 
 	}
 
+	public static class Indexes {
+
+		private AutoIndexMode auto = AutoIndexMode.NONE;
```

Comment:
I appreciated the creation of a namespace but that leads to a weird property name. Hibernate has a `ddl-auto` that's quite explicit. `spring.data.neo4j.auto-index` is interesting if we forget about the two other property.